platform/rpc/client.go

```diff
@@ -91,6 +91,20 @@ func (p *RPCClient) Version() (string, error) {
 	return version, err
 }
 
+// Export is used to get service configuration in platform-specific format
+func (p *RPCClient) Export(ids []flux.ServiceID) (string, error) {
+	var config string
+	err := p.client.Call("RPCServer.Export", ids, &config)
+	if _, ok := err.(rpc.ServerError); !ok && err != nil {
+		return "", platform.FatalError{err}
+	} else if err != nil && err.Error() == "rpc: can't find method RPCServer.Export" {
+		// "Export" is not supported by this version of fluxd (it is old). Fail
+		// gracefully.
+		return "unknown", nil
```

Comment:
Yes, it probably should. Version is often made a special case (usually because it's thought of after the fact, as it was here; and sometimes because it's used for control flow, which it isn't here). But it'd be cleaner to do it as a protocol change, I think.